cd/manager/jobmanager/jobManager.go

```diff
@@ -360,7 +377,17 @@ func (m *JobManager) processAnchorJobs(dequeuedJobs []manager.JobState) bool {
 			log.Printf("processAnchorJobs: starting anchor job: %s", manager.PrintJob(anchorJob))
 			m.advanceJob(anchorJob)
 		}
-		return len(dequeuedAnchors) > 0
+		// If not enough anchor jobs were queued by the scheduler to meet the configured minimum number of workers, add
```

Comment:
minimum should be ignored if scheduler is queueing. we need one source of truth.

Reply:
The scheduler already isn't the source of truth - we don't honor it's requests to spawn workers if there is more than 1 running, even today.